### DISCUSSION TITLE
Why OpenAI obfuscated the output

### DISCUSSION THREAD

**Discussion by daveshap** [daveshap/Raspberry#5] [Research]
![image](https://github.com/user-attachments/assets/da2b9bb5-f45d-49ce-928d-0af88d61be4b)

Consider:

1. Noam says it's all trained to do it one shot, this lends credence to the idea that it has <private> tags or <thinking> tags or something like that. 
2. There was a similar post that said that it had access to an unfettered, unfiltered model to do more open-ended thinking. I will see if I can find it.

**Comment by russellballestrini** [comment]
Our open version will show the chain and chain summary as a click to display above the LLM completion.

**Reply by daveshap** [reply]
No it will just all be in the open, no reason to obfuscate anything.

**Reply by russellballestrini** [reply]
Maybe not hidden but The chains itself might not be very useful for multi turn convos. Especially incorrect chains could likely poison the context window if not handled separately from `solutions`.

**Reply by daveshap** [reply]
make sure you watch the video, I explain the reasoning and show how it should look cleaned up

**Reply by russellballestrini** [reply]
Understood. I'm talking about an app or frontend for the resulting model during inference, not the finetuning process.

For example Claude 2 was notorious for context poisoning due to its censored nature.

Watching now at 1x speed. ; )

**Reply by daveshap** [reply]
we are not going to be working on a public API, frontend, or web app. We are building a dataset for finetuning a model. Please make sure you understand the purpose of this project and do not try to change the purpose or scope.

**Reply by blackeuler** [reply]
Will the fine-tune be sufficient to mimic the behavior? I think your data approach makes sense but Im thinking they have to be using the Stanford Self Reasoner paper as well. https://arxiv.org/abs/2203.14465
The paper says that fine-tune's couldn't match the performance.

**Reply by daveshap** [reply]
we will find out!
